<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>Abstract</title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>



<!-- MathJax scripts -->
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>


<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}
pre {
  overflow-x: auto;
}
pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
</style>



</head>

<body>
<p>Antibiotic induced alterations of the murine gut microbiota and
subsequent effects on colonization resistance against <em>Clostridium
difficile</em></p>

<p><strong>Running title:</strong> Colonization resistance against <em>C. difficile</em></p>

<p>Alyxandria M. Schubert, Hamide Sinani, and Patrick D. Schloss*</p>

<p>Correspondence:</p>

<blockquote>
<p>Department of Microbiology and Immunology
University of Michigan
1520A Medical Science Research Bldg. I
1500 W. Medical Center Dr.
Ann Arbor, MI 48109
<a href="mailto:pschloss@umich.edu">pschloss@umich.edu</a>
734.647.5801</p>
</blockquote>

<h3>Abstract</h3>

<p>Perturbations to the gut microbiota can result in a loss of colonization resistance against gastrointestinal pathogens such as <em>Clostridium difficile</em>. Although <em>C. difficile</em> infection is commonly associated with antibiotic use, the precise alterations to the microbiota associated with this loss in function are unknown. We used a variety of antibiotic perturbations to generate a diverse array of gut microbiota structures, which were then challenged with <em>C. difficile</em> spores. Across these treatments we observed that <em>C. difficile</em> resistance was never attributable to a single organism, but rather it was the result of multiple microbiota members interacting in a context-dependent manner. Using relative abundance data, we built a machine learning regression model to predict the levels of <em>C. difficile</em> that were found 24 hours after challenging the perturbed communities. This model was able to explain 77.2% of the variation in the observed number of <em>C. difficile</em> per gram of feces. This model revealed important bacterial populations within the microbiota, which correlation analysis alone did not detect. Specifically, we observed that populations associated with the <em>Porphyromonadaceae</em>, <em>Lachnospiraceae</em>, <em>Lactobacillus</em>, and <em>Alistipes</em> were protective and populations associated with <em>Escherichia</em> and <em>Streptococcus</em> were associated with high levels of colonization. In addition, a population affiliated with <em>Akkermansia</em> indicated a strong context dependency on other members of the microbiota. Together, these results indicate that individual bacterial populations do not drive colonization resistance to <em>C. difficile</em>. Rather, multiple diverse assemblages act in concert to mediate colonization resistance.</p>

<h3>Importance</h3>

<p>The gastrointestinal tract harbors a complex community of bacteria, known as the microbiota, which plays an integral role preventing its colonization by gut pathogens. This resistance has been shown to be crucial for protection against <em>C. difficile</em> infections (CDI), which are the leading source of hospital-acquired infections in the United States. Antibiotics are a major risk factor for acquiring CDI due to their effect on the normal structure of the indigenous gut microbiota. We found that diverse antibiotic perturbations gave rise to altered communities that varied in their susceptibility to <em>C. difficile</em> colonization. Instead of being dependent on one specific population of bacteria, we found that multiple co-existing populations conferred resistance. By understanding the relationships between <em>C. difficile</em> and members of the microbiota it will be possible to better manage this important infection.</p>

<h3>Introduction</h3>

<p>The microbiota, or the diverse community of microorganisms living in and
on the body, has an integral role in deterring pathogen colonization and
infection {Vollaard, 1994 #1}. This native protection by the microbiota
from invasive pathogenic species is termed colonization resistance. It
is well established that the gut bacterial microbiota is critical in the
host&#39;s defense against the pathogen <em>Clostridium difficile</em> {Chang, 2008
#3;Reeves, 2011 #4;Wilson, 1985 #2}. Perturbations to this indigenous
community often lead to a loss of resistance. This is especially
important in many hospital settings where patients are not only exposed
to various types and degrees of perturbations, such as antibiotics, diet
changes, and chemotherapy, but they are also exposed to <em>C. difficile</em>
spores from their environment {Bauer, 2015 #5}. <em>C. difficile</em>
infections (CDI) are the most reported hospital-acquired infection in
the United States and are responsible for 14,000 deaths a year {Lessa,
2015 #6}.</p>

<p>It is not completely understood how different perturbations to the gut
microbiota result in a loss of colonization resistance to <em>C.
difficile</em>. There is a clear need to better understand the ecology of
<em>C. difficile</em> and its interactions with members of the microbiota. In
mouse models of CDI, the unperturbed, untreated murine microbiome is
completely resistant to <em>C. difficile</em> colonization. It was previously
shown that C57Bl/6 mice treated with cefoperazone {Reeves, 2011
#4;Theriot, 2011 #7}, tigecycline {Bassis, 2014 #8}, clindamycin
{Buffie, 2012 #9}, or clindamycin in combination with a five antibiotic
cocktail {Chang, 2008 #3} had decreased colonization resistance. These
studies suggest that a loss of <em>Lachnospiraceae</em> and <em>Barnesiella</em> and a
bloom of <em>Lactobacillaceae</em> and <em>Enterobacteriaceae</em> are responsible for
the loss of colonization resistance. These results are largely supported
by human association studies {Schubert, 2014 #11;Vincent, 2013 #10}.
We previously observed significant differences between the gut microbiota of
hospitalized individuals with and without <em>C. difficile</em> and
non-hospitalized controls {Schubert, 2014 #11}. In addition, fecal
microbiota transplants have been shown to increase the relative abundance of <em>Bacteroidetes</em> and decrease the relative abundance of <em>Proteobacteria</em> and result in a successful
restoration of colonization resistance in patients {Seekatz, 2014 #12}.
The mechanisms involved in restoring colonization resistance are not fully understood, but this treatment emphasizes the importance of the overall gut microbiota in protecting against <em>C. difficile</em>.</p>

<p>Because the gut microbiota is a complex community we need tools that
enable us to dissect the interactions within the community and with <em>C.
difficile</em>. One approach is the use of mathematical models to identify
associations between members of the microbiota and <em>C. difficile</em>.
Mathematical models have been used to predict <em>C. difficile</em> {Schubert,
2014 #11;Stein, 2013 #13} and <em>Citrobacter</em> infection {Belzer, 2014
#14}, colon cancer {Zackular, 2013 #15}, and psoriasis {Statnikov,
2013 #16} based on the composition of the gut microbiota. We similarly
sought to identify the subset of the normal murine microbiota that are
responsible for colonization resistance by using mathematical models to
explain the relationship between members of the gut microbiota.</p>

<p>The purpose of this investigation was to expand our current knowledge of
the effects of various perturbations on colonization resistance against
<em>C. difficile</em>. Through the administration of different antibiotic
classes, doses, and recovery times we altered the murine gut microbiota
and challenged the communities with <em>C. difficile</em> spores to quantify
differences in colonization resistance. We then used 16S rRNA gene
sequencing to identify structural changes within the microbiota that
would be predictive of colonization resistance. Using these data, we
built a random forest regression model to predict <em>C. difficile</em> colonization
levels. Through this analysis, we have identified groups of related
bacteria that are associated with <em>C. difficile</em> colonization
resistance. This model revealed that the interactions giving rise to
colonization resistance were non-linear and context dependent. These
findings show we can successfully apply modeling techniques to accurately
predict the colonization resistance of a given microbiota..</p>

<h3>Results</h3>

<p><strong>Antibiotics differentially alter the structure of the microbiota and their colonization resistance to</strong> <strong><em>C. difficile.</em></strong> We selected a panel of seven antibiotics from six classes with the goal of differentially altering the microbiota and assessing their resistance to <em>C. difficile</em> colonization (<strong>Table 1</strong>). Following the cessation of antibiotics, each treatment group was given one day of recovery prior to challenge with <em>C. difficile</em> spores. One day post challenge we enumerated the density of <em>C. difficile</em> in the animals&#39; feces. We observed reproducibly high levels of <em>C. difficile</em> colonization in mice treated with cefoperazone, metronidazole, and streptomycin (<strong>Figures 1 and S1</strong>). We observed the most variance in levels of <em>C. difficile</em> colonization in mice treated with ampicillin. None of the mice that received ciprofloxacin were colonized. In addition to administering ciprofloxacin by oral gavage, we provided ciprofloxacin by intraperitoneal injection (10 mg/mL). For both approaches we provided one or two days of recovery from the antibiotic treatment. Regardless of the method, the resulting communities were resistant to <em>C. difficile</em> colonization. Only one of six mice receiving vancomycin was colonized with <em>C. difficile.</em> We suspected that this was due to residual vancomycin repressing <em>C. difficile</em> growth. In fact, two days post <em>C. difficile</em> challenge, <em>C. difficile</em> bloomed in this treatment group to a median of 9.1x10<sup>7^</sup> (interquartile range 7.6x10<sup>7<sup>–1.1x10<sup>8<sup>)</sup></sup></sup></sup> CFU/g feces. Furthermore, given two days of post-vancomycin recovery, there was no delay in <em>C. difficile</em> colonization to high levels, and on day one post challenge we observed a median of 3.0x10<sup>7^</sup> (interquartile range 2.6x10<sup>7<sup>–3.6x10<sup>7<sup>,</sup></sup></sup></sup> N=4) CFU/g feces. These results suggest that although the gut tissue does not absorb vancomycin, the absence of <em>C. difficile</em> in the remaining five vancomycin-treated mice may have been due to residual antibiotics lingering in the environment. Overall, these various antibiotic perturbations provided varying levels of colonization by <em>C. difficile,</em> which led us to hypothesize that the resulting communities varied in their composition.</p>

<p>To test this hypothesis, we sequenced the 16S rRNA genes from the fecal communities of treated and untreated mice prior to <em>C. difficile</em> challenge to identify populations within the microbiota that conferred colonization resistance. All of the antibiotic treatments, except for the ciprofloxacin-treated mice (AMOVA, P=0.09), resulted in distinct and reproducible changes to the structure of the microbiota relative to the untreated animals (AMOVA, P&lt;0.001). The similarity in the structure of the microbiota in ciprofloxacin-treated and untreated mice suggests that a higher dose of ciprofloxacin may have been necessary to significantly perturb the microbiota to allow <em>C. difficile</em> to overcome colonization resistance. Comparisons of the microbiota between antibiotic classes indicated that their structures were significantly different from each other (AMOVA, P&lt;0.03). The community structures of mice receiving beta-lactams (i.e. cefoperazone and ampicillin) were not significantly different from each other (AMOVA, P=0.37). These results indicate that perturbing the gut microbiota with antibiotics resulted in non-overlapping community structures that yielded significant variation in susceptibility to colonization when challenged with <em>C. difficile</em>.</p>

<p><strong>Reduced perturbations result in altered levels of colonization.</strong> Based on the <em>C. difficile</em> colonization levels in our seven antibiotic treatments, we hypothesized that titrating the dose of antibiotics that the mice received would result in smaller perturbations to the microbiota. Consequently we expected a greater maintenance of resistance against <em>C. difficile</em> colonization in these titrated treatment groups. In addition to the previous treatments, we treated mice with lower concentrations of cefoperazone, streptomycin, and vancomycin  (<strong>Figure S2</strong>). These antibiotics were selected because they are thought to target a broad spectrum of bacteria (i.e. cefoperazone), Gram-negative (i.e. streptomycin), and Gram-positive (i.e. vancomycin) bacteria. As expected in all mice receiving titrated doses of cefoperazone, <em>C. difficile</em> colonization levels decreased significantly (P&lt;0.02; <strong>Figure 2</strong>). Titrating the dose of cefoperazone in the animals&#39; drinking water resulted in significant decreases in the relative abundance of an OTU associated with the genus <em>Escherichia</em> (OTU 3) and a number of rare OTUs. We also observed increases in the relative abundances of OTUs associated with the family <em>Porphyromonadaceae</em> (OTU 5, 10, 11, 13, and 21; <strong>Figure 2</strong>). Reducing the dose of streptomycin significantly reduced the colonization levels (P&lt;0.01; <strong>Figure 2</strong>). Titrating the dose of streptomycin in the drinking water resulted in significant changes in the relative abundance of OTUs associated with the <em>Porphyromonadaceae</em> (OTUs 2, 5, 6, 10, and 11), <em>Alistipes</em> (OTU 12), and <em>Bacteroidales</em> (OTU 17). In addition to its anti-Gram-positive activity, vancomycin was also selected because although the community was quite different from untreated mice, we observed high levels of <em>C. difficile</em> colonization in only one mouse. We anticipated that lower doses might result in a community structure that would result in colonization. In fact, the 0.3 and 0.1 mg/mL doses of vancomycin resulted in similarly high levels of <em>C. difficile</em> colonization (P=0.96). Seven OTUs were differentially represented across the three doses of vancomycin. Surprisingly, even though the colonization levels of <em>C. difficile</em> did not significantly differ between the mice receiving 0.1 and 0.3 mg/mL of vancomycin in their drinking water, four of the OTUs that had significantly different relative abundances were only found in the lower dose. Three of these were affiliated with members of the <em>Porphyromonadaceae</em> (OTUs 2, 5, and 6) and one was affiliated with a member of the genus <em>Bacteroides</em> (OTU 1). Two OTUs affiliated with the <em>Akkermansia</em> (OTU 6) and <em>Lactobacillus</em> (OTU 8) genera increased with increasing dose and a third OTU affiliated with <em>Escherichia</em> (OTU 4) had a mixed response to the dose level. These results suggest that the context in which specific members of the microbiota are found is important in determining the overall resistance to <em>C. difficile</em>. For example, the relationship between the <em>Bacteroides</em>-affiliated OTU (OTU 1) and <em>C. difficile</em> colonization was positive in streptomycin-treated mice and it was negative in cefoperazone-treated mice. In addition, cefoperazone and streptomycin-treated mice had high levels of <em>C. difficile</em> although the former had significantly higher levels of an <em>Escherichia</em>-affiliated OTU (OTU 3), which were absent in the streptomycin-treated mice. Together, these results suggest that individual populations were not sufficient to consistently predict colonization resistance. In light of such results, resistance is likely a product of the overall composition of the community.</p>

<p><strong>Allowing recovery of the microbiota restores colonization resistance.</strong> In the experiments we have described thus far, we allowed the gut microbiota to recover for 24 hours before challenging them with <em>C. difficile</em>. Several studies have demonstrated that perturbed communities can return to a &ldquo;healthy&rdquo; state in which resistance to <em>C. difficile</em> is restored {Reeves, 2011 #118; Theriot, 2014 #956}. To test the effect of recovery on colonization and gain greater insights into the populations that confer colonization resistance, we allowed the microbiota of the mice that received the full metronidazole and ampicillin treatment to recover for an additional five days (<strong>Figure S3</strong>). Among the metronidazole-treated mice, those with extended recovery had a \(1.86e+06\)-fold reduction in colonization (P&lt;0.001; <strong>Figure 3</strong>). In addition, 7 of the 14 mice given the longer recovery period had no detectable <em>C. difficile</em> 24 hours after challenge. We detected six OTUs that were differentially represented in the two sets of metronidazole-treated mice (<strong>Figure 3</strong>). Most notable among these were two OTUs that affiliated with a member of the <em>Barnesiella</em> (OTU 2) and the <em>Escherichia</em> (OTU 3). The relative abundance of this <em>Barnesiella</em>-affiliated OTU increased with the delay, and the relative abundance of this <em>Escherichia</em>-affiliated OTU decreased. Similar to the metronidazole-treated mice, the ampicillin-treated mice that were allowed to recover an additional five days before challenge had a significant decrease in <em>C. difficile</em> colonization (P=0.03). As before, we observed a similar increase and decrease in relative abundances for <em>Barnesiella</em> (OTU 2) and <em>Escherichia</em> (OTU 3)-affiliated OTUs. However untreated, fully resistant mice harbored significantly lower levels of the <em>Barnesiella</em>-affiliated OTU (OTU 2). Rather, untreated mice had high levels of various <em>Porphyromonadaceae</em>-affiliated OTUs (<strong>Figure 1</strong>). These findings further confirm the context-dependency of colonization resistance suggested by the results of our titration experiments.</p>

<p><strong>Correlation analysis reveals potentially protective bacteria.</strong> To identify bacterial taxa that could be associated with resistance or susceptibility to <em>C. difficile</em> across the three sets of experiments, we measured the correlation between the relative abundance of each OTU on the day of inoculation with the level of <em>C. difficile</em> colonization 24 hours later (<strong>Figure 4</strong>). OTUs associated with providing resistance against <em>C. difficile</em> (N=22) outnumbered those with associated with susceptibility (N=9). The <em>Porphyromonadaceae</em>-affiliated OTUs (ρ<sub>average</sub>=-0.52, N=11 OTUs) were consistently associated with low levels of <em>C. difficile</em> colonization. Among the three <em>Proteobacteria</em>-affiliated OTUs with a significant positive association with <em>C. difficile</em> colonization, the strongest was affiliated with the <em>Escherichia</em> (OTU 3; ρ=0.54). By performing an OTU-based analysis we were able to observe intra-family and genus differences in association with <em>C. difficile</em> colonization. For example, the <em>Lachnospiraceae</em> have been associated with protection against <em>C. difficile</em>. Although within the <em>Lachnospiraceae</em> family we observed three OTUs that were associated with low levels of <em>C. difficile</em> colonization, one OTU was associated with high levels of <em>C. difficile</em>. In addition we observed three significantly correlated <em>Lactobacillus</em>-affiliated OTUs (family <em>Lactobacillaceae</em>, two) of which were associated with low levels of <em>C. difficile</em> and one was associated with high levels. The broad taxonomic representation of OTUs associated with low levels of <em>C. difficile</em> again suggests that a diverse community may be advantageous in preventing <em>C. difficile</em> colonization.</p>

<p><strong>The composition of the disturbed gut microbiota is predictive of</strong> <strong><em>C. difficile</em></strong> <strong>colonization levels.</strong> These three sets of experiments demonstrated that in certain contexts individual OTUs could be associated with <em>C. difficile</em> colonization, but in other contexts those OTUs had the opposite or no association. This suggests that colonization is a phenotype that is driven by multiple populations that act independently and possibly in concert to resist colonization. Correlation-based analyses cannot predict these types of context dependencies because they do not take into account the non-linearity and statistical interactions between populations. Therefore, we used a regression-based random forest machine learning algorithm to predict the level of <em>C. difficile</em> colonization observed in the three sets of experiments using the composition of the microbiota at the time of challenge as predictor variables. The model explained 77.2% of the variation in the observed <em>C. difficile</em> colonization levels (<strong>Figure 5</strong>). When we only included the top 12 OTUs based on the percent increase in the mean squared error when each OTU was removed, the resulting model explained 77.1% of the variation in the observed <em>C. difficile</em> colonization levels. The OTUs that were ranked as being the most important in defining the random forest model further validated the observations from the correlation-based analysis (<strong>Figure S4</strong>). According to the random forest model, colonization resistance was associated with OTUs that affiliated with the <em>Porphyromonadaceae</em> (OTU 15, 10, 6, 18, and 11), <em>Lachnospiraceae</em> (OTU 25), <em>Lactobacillus</em> (OTU 23), <em>Alistipes</em> (OTU 12), and <em>Turicibacter</em> (OTU 9; <strong>Figure 6</strong>). A loss in these populations, concurrently with a gain in OTUs affiliated with the <em>Escherichia</em> (OTU 3) or <em>Streptococcus</em> (OTU 90), was associated with an increased susceptibility to infection (<strong>Figure 6</strong>). As we observed in the titration experiments, the relationship between an <em>Akkermansia</em>-affiliated OTU (OTU 4) and <em>C. difficile</em> indicated that wide variation in the relative abundance of <em>Akkermansia</em> was associated with varying levels of <em>C. difficile</em>. There were varying abundances of the <em>Akkermansia</em>-affiliated OTU in mice regardless of the level of <em>C. difficile</em> colonization. Finally, as indicated by the number of OTUs with relative abundances below the limit of detection, those mice could harbor varying levels of <em>C. difficile</em>. These observations bolster the hypothesis that colonization resistance is context dependent.</p>

<h3>Discussion</h3>

<p>Previous attempts to study the role of the gut microbiota in colonization resistance against <em>C. difficile</em> infection have utilized a single perturbation to the community. Here, we used seven antibiotics from six classes that were given to mice in varying doses and with varying post-antibiotic recovery periods. The result was a combination of 15 different perturbations and the non-perturbed microbiota, which allowed us to generate distinct community profiles that displayed varying susceptibility to <em>C. difficile</em> colonization. Our findings demonstrated that colonization resistance was not driven by individual populations, but by a consortium of organisms. Others have demonstrated that <em>Barnesiella</em> or <em>Lachnospiraceae</em> are partially protective against <em>C. difficile</em> {Buffie, 2012 #9;Reeves, 2012 #18}. Although we observed similar results in a subset of our perturbations, by using a large number of perturbations, we were able to demonstrate that a varied collection of populations was important for complete colonization resistance. Thus colonization resistance can be described as an emergent property of the microbiome, in which individual bacterial populations integrated in a community contribute to the overall resistance to <em>C. difficile</em> {Novikoff, 1945 #34}.</p>

<p>There is clear need for more efficient therapies for treatment of <em>C. difficile</em> infections in humans aimed at restoration of the microbiota. Current first line treatments of CDI include regimens of either metronidazole or vancomycin, which further perturb the microbiota. As such, relapse rates for CDI are typically around 25-30% {Wilcox, 1998 #19}. Interestingly, we observed that treatment with either antibiotic induced susceptibility to <em>C. difficile</em> in mice. This result has implications for understanding the causes of recurrent infections. Previous efforts to restore the microbiota and reestablish colonization resistance also support our findings. For instance, association of germ-free mice with a <em>Lachnospiraceae</em> isolate only reduced the level of <em>C. difficile</em> colonization by 10 to 100-fold {Reeves, 2012 #18}. Using conventional mice, mixtures of bacteria rather than individual populations have been shown to restore colonization resistance and mediate clearance of <em>C. difficile</em> {Buffie, 2015 #21;Lawley, 2012 #20}. Fecal transplants, which represent a diverse collection of bacterial populations, have been highly effective in treating humans with recurrent <em>C. difficile</em> {Kassam, 2013 #22;Seekatz, 2014 #12;Weingarden, 2015 #23}. By generating a varied collection of communities that were challenged with <em>C. difficile</em>, we have identified a subset of populations using random forest modeling that could be used as a probiotic cocktail to provide colonization resistance. These would include members of the <em>Porphyromonadaceae</em>, <em>Lachnospiraceae</em>, <em>Lactobacillus</em>, and <em>Alistipes</em>. Several of these populations have been examined for their potential as a probiotic for preventing <em>C. difficile</em> infection. A 6-species cocktail, including isolates of <em>Porphyromonadaceae</em>, <em>Lachnospiraceae</em>, <em>Lactobacillus</em>, <em>Coriobacteriaceae</em>, <em>Staphylococcus</em>, and <em>Enterococcus</em>, successfully resolved CDI in mice {Lawley, 2012 #20}. In humans, <em>Lactobacillus</em>-based probiotics have been co-administered with antibiotics to deter the onset of antibiotic-associated diarrhea (AAD) and <em>C. difficile</em> infection {Lawley, 2012 #20}. A more diverse probiotic, which contained 33 bacterial species including <em>Porphyromonadaceae</em>, <em>Lachnospiraceae</em>, <em>Ruminococcaceae</em>, <em>Eubacteriaceae</em>, and <em>Lactobacillus</em> isolates, successfully restored colonization resistance in recurrent <em>C. difficile</em> infection and eliminated diarrhea up to 6 months post treatment {Petrof, 2013 #25}. Given this evidence, we feel confident that an effective probiotic mixture could be designed based on our findings to recover colonization resistance against <em>C. difficile</em>. Moreover, this line of study will be useful towards the development of personalized treatments based on an individual&#39;s specific gut microbiota, which may be a more efficient strategy for preventing and treating CDI. Further examination of the bacterial populations identified in this study is necessary to identify causal relationships and assess the specific mechanisms of colonization resistance. Additional investigations into the importance of taxonomic diversity of the microbiota in colonization resistance may highlight its underlying connection to functional redundancy, which is hypothesized to contribute to overall functional stability of the community, known as the &ldquo;insurance hypothesis&rdquo; {Yachi, 1999 #38}. Such research will further advance the development of protocols to prevent and treat CDI.</p>

<p>Random forest regression models allowed us to describe community resistance as a byproduct of an assemblage of bacterial populations rather than as individual populations. A correlation-based analysis was unable to identify populations that had a context dependent or non-linear association with <em>C. difficile</em> colonization. Although the murine and human microbiota do not fully overlap, our previous analysis of humans infected with <em>C. difficile</em> supports the populations that we associated with colonization {Schubert, 2014 #11}. For instance, <em>Escherichia</em> was previously associated with infected individuals and <em>Lachnospiraceae</em>, <em>Ruminococcaceae</em>, and <em>Alistipes</em> were absent from infected individuals. The overlap between the results from the current study and past human studies along with the power of random forest models suggest that it should be possible to model a patient’s risk of developing a <em>C. difficile</em> infection based on their gut microbiota composition at admission. As a demonstration of this, we generated a random forest model to predict the binary outcome of whether a mouse would become colonized, regardless of <em>C. difficile</em> abundance. With this model, which used the same OTUs as the regression model, we observed an error rate of 10.7%. This suggests that such an approach would be valuable and could perhaps be improved by incorporating other clinical data {Schubert, 2014 #11}. Overall these findings demonstrate the significance of the microbiota as an interconnected bacterial community in assessing resistance to pathogen colonization.</p>

<h3>Materials and Methods</h3>

<p><strong>Animal care.</strong> We used 5-8 week old C57Bl/6 mice obtained from a single breeding colony maintained at the University of Michigan for all of our experiments. These mice were reared under SPF conditions within the animal facility at the University of Michigan. All animal-related protocols and experiments were approved by the University Committee on Use and Care of Animals at the University of Michigan and carried out in accordance with the approved guidelines.</p>

<p><strong>Antibiotic administration.</strong> Mice were administered one of seven different antibiotics including cefoperazone, vancomycin, metronidazole, streptomycin, ciprofloxacin, ampicillin, and clindamycin (<strong>Table 1</strong>). The route of administration depended on the specific antibiotic. Cefoperazone (0.5, 0.3, or 0.1 mg/ml), vancomycin (0.625, 0.3, or 0.1 mg/ml), streptomycin (5, 0.5, or 0.1 mg/ml), metronidazole (0.5 mg/ml), and ampicillin (0.5 mg/ml) were all administered in the mouse drinking water for 5 days. Ciprofloxacin (10 mg/kg) was administered via oral gavage and clindamycin (10 mg/kg) was administered via intraperitoneal injection. Mice that had not received antibiotics were used as negative controls for these experiments because <em>C. difficile</em> is unable to colonize mice that are not perturbed by antibiotics. The &ldquo;No antibiotics&rdquo; group in Figures 1 and S1 collectively refer to the microbiota of these untreated animals prior to their challenge with C. difficile, as well as the microbiota of animals prior to treatment with their respective antibiotic (i.e. the baseline for antibiotic treated mice).</p>

<p><strong><em>C. difficile</em></strong> <strong>preparation and challenge.</strong> All antibiotic-treated mice were given 24 hours to recover with untreated drinking water prior to <em>C. difficile</em> challenge. <em>C. difficile</em> strain 630Δerm spores were used in all experiments. Spores were prepared from a single large batch whose concentration was determined within the week prior to each <em>C. difficile</em> challenge {Sorg, 2009 #26}. Spores were stored long term at 4°C. On the day of challenge 10<sup>3^</sup> <em>C. difficile</em> spores were administered to mice via oral gavage. Immediately following this challenge, the remaining <em>C. difficile</em> inoculum was diluted in a series and plated to confirm the correct dosage.</p>

<p><strong>Sample collection and plating.</strong> Fecal samples were freshly collected for each mouse on the day of <em>C. difficile</em> challenge. On the day after the challenge another fecal sample was weighed and diluted under anaerobic conditions with anaerobic PBS. The number of colony forming units (CFU) were counted following 24 hours growth on TCCFA plates at 37°C under anaerobic conditions {Buggy, 1983 #27}.</p>

<p><strong>DNA extraction and sequencing.</strong> Total bacterial DNA was extracted from each stool sample collected prior to challenge using the MOBIO PowerSoil®-htp 96 Well Soil DNA Isolation Kit. We generated amplicons of the V4 region within the 16S rRNA gene and sequenced the fragments using an Illumina MiSeq as previously described {Kozich, 2013 #28}.</p>

<p><strong>Sequence curation.</strong> These sequences were curated using mothur (v.1.35) as previously described {Kozich, 2013 #28;Schloss, 2009 #29}. Briefly, sequences were binned into operational taxonomic units (OTUs) using a 3% dissimilarity cutoff. Taxonomic assignments were determined by using a naïve Bayesian classifier with the Ribosomal Database Project (RDP) training set (version 10) requiring an 80% bootstrap confidence score {Wang, 2007 #1809}. In parallel to the fecal samples, we also sequenced a mock community where we knew the true sequence of the 16S rRNA gene sequences. Analysis of the mock community data indicated that the error rate following our curation procedure was 0.02%. All 16S rRNA gene sequence data and metadata are available through the Sequence Read Archive under accession SRP057386.</p>

<p><strong>Statistical analysis and modeling.</strong> Complete scripts for regenerating our analysis and this paper are available at the online repository for this study (<a href="https://github.com/SchlossLab/Schubert%5C_AbxD01%5C_mBio%5C_2015">https://github.com/SchlossLab/Schubert\_AbxD01\_mBio\_2015</a>). Comparisons between the antibiotic-treated communities were made by calculating dissimilarity matrices based on the metric of Yue and Clayton {Yue, 2005 #1061}. To avoid biases due to uneven sampling, the dissimilarity matrices were calculated by rarefying the samples to 1,625 sequences per sample. We then used analysis of molecular variance (AMOVA) to test for differences in community structure using 10,000 permutations {Anderson, 2001 #804}. OTU-based analyses were performed using R (v.3.1.2). After subsampling the OTU frequency data to 1,625 sequences per sample, OTUs were considered for analysis if their average relative abundance within any treatment group was at least 1% (N=38 OTUs). All OTU-by-OTU comparisons were performed using the Kruskal-Wallis rank sum test followed by pairwise Wilcoxon rank sum tests. Comparison of log (base 10) transformed <em>C. difficile</em> CFU/g feces between experimental groups was calculated using the Kruskal-Wallis rank sum test followed by pairwise Wilcoxon rank sum tests. Spearman rank correlation analysis was performed between OTU counts and <em>C. difficile</em> CFU/g feces. All P-values were corrected using a Benjamini and Hochberg adjustment with an experiment-wide Type I error rate of 0.05 {Benjamini, 1995 #604}. Random forest regression models were constructed using the randomForest R package using 10,000 trees {Cutler, 2007 #31}. To construct each tree, two-thirds of the samples were randomly selected were used to train the model and the other one-third of the samples were sampled with replacement to test the model. The regression was performed using the log (base 10) transformation of the number of CFU/g fecal material as the dependent variable and the 38 OTUs as predictor variables.</p>

<h3>Acknowledgements</h3>

<p>We thank Dr. Vincent Young for providing a critical review of an earlier version of this manuscript. This work was supported by several grants from the National Institutes for Health R01GM099514, R01HG005975, U19AI090871, and P30DK034933. The funding agencies had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>

<p><strong>Table 1. Description of Antibiotics used in this study</strong></p>

<table><thead>
<tr>
<th align="center">Antibiotic</th>
<th align="center">Administration</th>
<th align="center">Class</th>
<th align="center">Mechanism</th>
<th align="center">Target</th>
</tr>
</thead><tbody>
<tr>
<td align="center">Ampicillin</td>
<td align="center"><em>Ad libitum</em> in drinking water, five days</td>
<td align="center">β-lactam: Penicillin</td>
<td align="center">Inhibits peptidoglycan synthesis</td>
<td align="center">Gram +/-</td>
</tr>
<tr>
<td align="center">Cefoperazone</td>
<td align="center"><em>Ad libitum</em> in drinking water, five days</td>
<td align="center">β-lactam: Cephalosporin</td>
<td align="center">Inhibits peptidoglycan synthesis</td>
<td align="center">Gram +/-</td>
</tr>
<tr>
<td align="center">Ciprofloxacin</td>
<td align="center">Oral gavage, one time</td>
<td align="center">Fluoroquinolone</td>
<td align="center">Inhibits DNA gyrase</td>
<td align="center">Gram +/-</td>
</tr>
<tr>
<td align="center">Clindamycin</td>
<td align="center">Intraperitoneal injection, one time</td>
<td align="center">Lincosamide</td>
<td align="center">Inhibits protein synthesis</td>
<td align="center">Anaerobes</td>
</tr>
<tr>
<td align="center">Metronidazole</td>
<td align="center"><em>Ad libitum</em> in drinking water, five days</td>
<td align="center">Nitromidazole</td>
<td align="center">Destabilizes bacterial DNA</td>
<td align="center">Anaerobes</td>
</tr>
<tr>
<td align="center">Streptomycin</td>
<td align="center"><em>Ad libitum</em> in drinking water, five days</td>
<td align="center">Aminoglycoside</td>
<td align="center">Inhibits protein synthesis</td>
<td align="center">Gram +/-</td>
</tr>
<tr>
<td align="center">Vancomycin</td>
<td align="center"><em>Ad libitum</em> in drinking water, five days</td>
<td align="center">Glycopeptide</td>
<td align="center">Inhibits peptidoglycan synthesis</td>
<td align="center">Gram +</td>
</tr>
</tbody></table>

<h3>Figure Legends</h3>

<p><strong>Figure 1. Antibiotic treatments result in significant alterations to the structure of the microbiota and variation in colonization resistance.</strong> Bars indicate the median percent relative abundance of those selected OTUs from all treatment groups on the day of <em>C. difficile</em> challenge. Stars along the x-axis indicate those OTUs that were significantly different from untreated mice for that antibiotic treatment after correcting for multiple comparisons. The error bars indicate the interquartile range. The median level <em>C. difficile</em> colonization found 24 hours post microbiota sampling is plotted on the right for each treatment with error bars indicating the interquartile range. The dose of antibiotic and the number of animals used in each treatment group is indicated for antibiotic treatment group. The treatment groups are sorted according to level of <em>C. difficile</em> colonization.</p>

<p><strong>Figure 2. Titration of antibiotic perturbations results in altered community structures and</strong> <strong><em>C. difficile</em></strong> <strong>colonization resistance.</strong> Bars indicate the median percent relative abundance of those selected OTUs from all treatment groups on the day of <em>C. difficile</em> challenge. Stars along the x-axis indicate those OTUs that varied significantly across doses of the same antibiotic after correcting for multiple comparisons. The error bars indicate the interquartile range. The median level <em>C. difficile</em> colonization found 24 hours post microbiota sampling is plotted on the right for each treatment with error bars indicating the interquartile range. The number of animals used in each treatment group is indicated in the legend, which depicts the doses of each antibiotic that were used.</p>

<p><strong>Figure 3. Increasing the recovery time following antibiotic perturbation restores colonization resistance.</strong> Bars indicate the median percent relative abundance of those selected OTUs from all treatment groups on the day of <em>C. difficile</em> challenge. Stars along the x-axis indicate those OTUs that varied significantly between those mice that were allowed 1 or 6 days of recovery after correcting for multiple comparisons. The error bars indicate the interquartile range. The median level <em>C. difficile</em> colonization found 24 hours post microbiota sampling is plotted on the right for each recovery period and antibiotic with error bars indicating the interquartile range. The number of mice used in each treatment group is indicated above the <em>C. difficile</em> colonization data. The dose of each antibiotic is indicated next to the name of the antibiotic.</p>

<p><strong>Figure 4. Diverse taxonomic groups are associated with low levels of</strong> <strong><em>C. difficile</em></strong> <strong>colonization.</strong> Spearman correlation coefficients were calculated using the relative abundance of OTUs found on the day that mice were challenged with <em>C. difficile</em> spores and the amount of <em>C. difficile</em> observed 24 hours later. Only significant correlations are presented after correcting for multiple comparisons. OTUs are grouped by the taxonomic family and the letters in the parentheses correspond to the phylum that the taxa belong to. B: <em>Bacteroidetes</em>, F: <em>Firmicutes</em>, P: <em>Proteobacteria</em>, A: <em>Actinobacteria</em>, T: <em>Tenericutes</em>.</p>

<p><strong>Figure 5. Random forest regression model predicts</strong> <strong><em>C. difficile</em></strong> <strong>colonization levels based on the structure of the microbiota.</strong> The overall model explained 77.2% of the variation in the data. Each pane shows antibiotic treatment groups in color and the other points as gray circles. The panes are sorted by the level of <em>C. difficile</em> colonization when mice were treated with the highest dose of their respective antibiotic.</p>

<p><strong>Figure 6. Relationship between OTU relative abundance and</strong> <strong><em>C. difficile</em></strong> <strong>colonization levels indicates non-linearity and context-dependency.</strong> The 12 OTUs that resulted in the greatest change in percent mean squared error when removed from the random forest regression model are shown in each pane and together explain 77.1% of the variation in the data. The Spearman correlation value between that OTUs abundance and <em>C. difficile</em> levels are shown for each pane when the corrected P-value was significant. The color and symbols represent the same antibiotic dose and recovery period as in Figure 5.</p>

<p><strong>Figure S1. Effect of antibiotic perturbations on phylum-level representation of communities on day of</strong> <strong><em>C. difficile</em></strong> <strong>challenge.</strong> Bars depict the median relative abundance across mice within the treatment group and error bars indicate the interquartile range.</p>

<p><strong>Figure S2. Effect of titrated antibiotic treatments on phylum-level representation of communities on day of</strong> <strong><em>C. difficile</em></strong> <strong>challenge.</strong> Bars depict the median relative abundance across mice within the treatment group and error bars indicate the interquartile range.</p>

<p><strong>Figure S3. Effect of recovery period following antibiotic treatments on phylum-level representation of communities on day of</strong> <strong><em>C. difficile</em></strong> <strong>challenge.</strong> Bars depict the median relative abundance across mice within the treatment group and error bars indicate the interquartile range.</p>

<p><strong>Figure S4. The change in percent mean squared error when each OTU was removed from the random forest regression model.</strong> The top 12 OTUs with the highest percent increase in mean square error are denoted in red text and are depicted in Figure 6.</p>

</body>

</html>
